Add fortios path traversal credential grabber (cve-2018-13379) (#14518)
rapid7/metasploit-framework: documentation/modules/auxiliary/scanner/http/fortios_vpnssl_traversal_leak.md

FortiOS system file leak through SSL VPN via specially crafted HTTP resource requests.

A path traversal vulnerability in the FortiOS SSL VPN web portal may allow an unauthenticated
attacker to download FortiOS system files through specially crafted HTTP resource requests.

## Vulnerable Application

> This exploit read `/dev/cmdb/sslvpn_websession` file, this file contains login and passwords in (clear/text).

Reviewer: ```suggestion
This module reads logins and passwords in clear text from the `/dev/cmdb/sslvpn_websession` file.
```

This vulnerability affect (FortiOS 5.4.6 to 5.4.12, FortiOS 5.6.3 to 5.6.7 and FortiOS 6.0.0 to 6.0.4).

## Verification Steps

1. Start msfconsole
2. Do: use auxiliary/scanner/http/fortios_vpnssl_traversal_leak
3. Do: set RHOSTS [IP]
4. Do: set RPORT 10443
5. Do: run

## Options

### DUMP_FORMAT

Dump format. (Accepted: raw, ascii)

### STORE_CRED

Store credential into the Metasploit database.

### STORE_LOOT

Store dump in loot directory.

## Scenarios

### Usages

> You can scan and get all credentials on the remote target when you run the followind command:

Reviewer: ```suggestion
You can scan and get all credentials on the remote target when you run the following command:
```
